pkgs/tools/misc/direnv/default.nix

```diff
@@ -18,6 +18,11 @@ buildGoModule rec {
     stdenv.lib.optionalString (!stdenv.hostPlatform.isWindows)
     "${bash}/bin/bash";
 
+  # fix hardcoded GOFLAGS in makefile
+  postPatch = ''
+    substituteInPlace GNUmakefile --replace "export GOFLAGS=-mod=vendor" ""
+  '';
+
   # replace the build phase to use the GNUMakefile instead
   buildPhase = ''
     make BASH_PATH=$BASH_PATH
```

Comment:
```suggestion
    make BASH_PATH=$BASH_PATH GOFLAGS=""
```

Reply:
This breaks the build, would need to be `GOFLAGS="$GOFLAGS"`.

I'd rather keep the replace as I think it's clearer.

Reply:
ok.